app/src/main/java/org/fossasia/openevent/general/order/OrderDetailsViewHolder.kt

```diff
@@ -5,18 +5,7 @@ import android.net.Uri
 import android.provider.CalendarContract
 import android.view.View
 import androidx.recyclerview.widget.RecyclerView
-import kotlinx.android.synthetic.main.item_card_order_details.view.calendar
-import kotlinx.android.synthetic.main.item_card_order_details.view.date
-import kotlinx.android.synthetic.main.item_card_order_details.view.eventDetails
-import kotlinx.android.synthetic.main.item_card_order_details.view.eventName
-import kotlinx.android.synthetic.main.item_card_order_details.view.eventSummary
-import kotlinx.android.synthetic.main.item_card_order_details.view.location
-import kotlinx.android.synthetic.main.item_card_order_details.view.map
-import kotlinx.android.synthetic.main.item_card_order_details.view.name
-import kotlinx.android.synthetic.main.item_card_order_details.view.orderIdentifier
-import kotlinx.android.synthetic.main.item_card_order_details.view.organizer
-import kotlinx.android.synthetic.main.item_card_order_details.view.qrCodeView
-import kotlinx.android.synthetic.main.item_card_order_details.view.organizerLabel
+import kotlinx.android.synthetic.main.item_card_order_details.view.*
```

Comment:
don't use `WildCard` imports . That fails your spotlesscheck

Reply:
Revert the wildcard import